packages/ui/src/containers/PasscodeValidation/index.tsx

```diff
@@ -102,7 +102,7 @@ const PasscodeValidation = ({ type, method, className, target }: Props) => {
     <form className={classNames(styles.form, className)}>
       <Passcode
         name="passcode"
-        className={classNames(styles.field, error && styles.withError)}
+        className={classNames(styles.inputField, error && styles.withError)}
```

Comment:
nit: ditto. belongs to another PR?

Reply:
minor update as above, I'll split it next time.